<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录到聊天室：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Username_Text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/image/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>501</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>尚未登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>104</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="CreateUser">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>没有账户？注册一个！</string>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>Username_Text</zorder>
  <zorder>Password_Text</zorder>
  <zorder>login_label</zorder>
  <zorder>Login_Button</zorder>
  <zorder>CreateUser</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
